conda-store-server/conda_store_server/build.py

```diff
@@ -155,6 +158,29 @@ def build_environment(conda_command, environment_filename, conda_prefix):
     )
 
 
+def set_conda_environment_variables(
```

Comment:
Is this required? It looks like it is built into it https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html#setting-environment-variables. I think all that is needed is https://github.com/Quansight/conda-store/pull/424/files#diff-127657f20e6c5c5301f08bded4596eca413c74ef25a9d0f5c1e12fada1bfe16cR191 for this PR.

Reply:
Hmmm - I recall trying it without this function because I thought it would be handled by conda, but no environment variables were found or activated!

Reply:
I'll try it and report back shortly!

Reply:
And if not we should be using this file instead https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html#macos-and-linux

Reply:
It is using those files :) trying it without this method, I wasn't able to get vars set.

Reply:
It is using those files :) trying it without this method, I wasn't able to get vars set.